chart/templates/proxy-deployment.yaml

```diff
@@ -131,7 +131,7 @@ spec:
 {{ include "gitpod.container.defaultEnv" (dict "root" . "gp" $.Values "comp" $wsProxy "noVersion" true) | indent 8 }}
         volumeMounts:
 {{- if (and $wsProxy (not $wsProxy.disabled) $wsProxy.portRange) }}
-        - name: ws-proxy-config
+        - name: {{ template "gitpod.comp.configMap" $this }}
```

Comment:
Seems like this line caused a breakage, because the new name becomes `proxy-config` (instead of `ws-proxy-config`), because of this above:

```yml
  template:
    metadata:
      name: proxy
```

Reply:
Oh, maybe it should be `$thisWsProxy` instead of `$this`? Cc @csweichel